2015/10-dancer-i18n-with-log-report.pod

```diff
@@ -0,0 +1,206 @@
+
+ use Log::Report::Translator::POT;
+
+ (textdomain 'my-domain')->configure(
+   translator => Log::Report::Translator::POT->new(
+     lexicon => 'messages' # Directory of the PO files
+   )
+ );
+
+Now run your application, this time specifying another language:
+
+ LC_MESSAGES=de_DE.utf8 perl bin/app.pl
+
+When you visit the index page, the text should now have been appropriately
+translated.
+
+=head2 Translating text within templates
+
+Translating text within templates is very similar to translating strings within
+the application, except that being a template, the translation functions aren't
+quite so readily available. For the purposes of this example, I'm going to
+assume that you're using C<Template::Toolkit>.
+
+As we don't have a readily available translating function, we'll do things the
+other way around this time, and start by providing a translating function to each
+template. Add the following to your route handler:
+
+ sub translate {
+   my $msg = Log::Report::Message->fromTemplateToolkit('tt-domain', @_);
+   $msg->toString;
+ }
+ 
+ hook before_template => sub {
+   shift->{loc}  = \&translate;
+ };
+
+The above code adds a subroutine C<loc> that can be called from within each
+template. Note that we are using a different domain. This is normally best so
+as to keep the sets of messages completely separate, but it's also possible to
+use the same domain if you prefer.
+
+ As a simple example, add the following to your index template:
+
+ [% loc("We are dancing") %]
+
+(See L<Log::Report::Extract::Template> for more powerful examples.)
+
+Again, we'll need to extract the message IDs for translation. This can be done
+in a similar manner to last time, but this time specifying template files:
+
+ xgettext-perl -p messages --from-code=iso-8859-1 --template TT2-loc --domain tt-domain .
+
+Note that if you are using the same domain, then this will remove message
+strings from your original extractions (unless the same messages appear in the
+template files). To prevent this, you can you use the C<--no-cleanup> option,
+in which case the previous messages will be commented out. As per the comment
+above, it's best to use separate domains for this reason.
+
+Repeat the previous process, copying the default translation to a German file
+in the new domain, and add the translation:
+
+ mkdir messages/tt-domain/
+ cp messages/tt-domain.utf-8.po messages/tt-domain/de_DE.utf8.po
+
+ msgid "We are dancing"
+ msgstr "Wir tanzen"
+
+Finally add a new translator for the new domain:
+
+ (textdomain 'tt-domain')->configure(
+   translator => Log::Report::Translator::POT->new(
+     lexicon => 'messages' # Directory of the PO files
+   )
+ );
+
+Re-run the application, and both the previous string as well as new template
+text should be translated.
+
+=head2 Handling plurals
+
+I won't go into the full power of L<Log::Report>, but to give a flavour of
+what's possible, here's how to handle pluralisation:
```

Comment:
"Give a flavour" is repeated three times in both articles, and in the end at both articles. You might want to rephrase some to avoid repetition.

"Here is an example of pluralisation
[...]
(You can read more about it here.)"

Might be easier.